.eslintrc.yml

```diff
@@ -48,8 +46,13 @@ overrides:
     parser: '@typescript-eslint/parser'
     rules:
       # Argh.
-      '@typescript-eslint/explicit-function-return-type': 'off'
+      '@typescript-eslint/explicit-function-return-type':
+        ['error', { 'allowExpressions': true }]
+      '@typescript-eslint/no-empty-function': 'error'
+      '@typescript-eslint/no-var-requires': 'error'
```

Comment:
If these are fixed, maybe we can remove these rules and use the default configuration?

Reply:
Because these rules throw errors on .js files, they are turned off at the top level, so I think they need to be turned back on for .ts files.